Arrowgene.Ddon.GameServer/Characters/WalletManager.cs

```diff
@@ -17,6 +19,26 @@ public class WalletManager
 
         private IDatabase _Database;
 
+        private static readonly Dictionary<WalletType, uint> WalletLimits = new()
+        {
+            {WalletType.Gold, 999999999},
+            {WalletType.RiftPoints, 999999999},
+            {WalletType.BloodOrbs, 50000},
```

Comment:
If we change this to say 100k, does the game stop making the warning about the BO currency until 100k?

Reply:
Everything should be a server side cap. This is an MMO. Probably at some point all these settings should be configured via the server config. I could imagine someone might be interested in adjusting the exp scale for example. Probably doesn't need to be immediately done though.